templates/react/components/foo/Show.js

```diff
@@ -0,0 +1,97 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {Link} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {retrieve, reset} from '../../actions/{{{ lc }}}/show';
+import { del, loading, error } from '../../actions/{{{ lc }}}/delete';
+
```

Comment:
done in this commit https://github.com/mysiar/generate-crud/commit/acd90debcf72cf1b73aeef1c4faee0ec860fe808 
but for some reason change is not reflected above